quinn-proto/Cargo.toml

```diff
@@ -17,8 +17,9 @@ all-features = true
 maintenance = { status = "experimental" }
 
 [features]
-default = ["tls-rustls", "log"]
-tls-rustls = ["rustls", "ring"]
+default = ["tls-rustls", "rustls-ring", "log"]
+tls-rustls = ["rustls-pki-types"]
+rustls-ring = ["rustls/ring", "ring"] # Allow rustls to be used with other crypto providers
```

Comment:
This wording makes it seem like you should *enable* the feature if you want that.
```suggestion
# Disable to allow rustls to be used with other crypto providers enabled elsewhere
rustls-ring = ["rustls/ring", "ring"]
```